notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>"We use Langchain's <code>RecursiveCharacterTextSplitter</code>, which makes efficient use of code language detection to make better splits the lenght function we use is not the count of characters, but the count of tokens. " - I don't understand this sentence, maybe it's two sentences? Also, there's a typo. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>